doc/whats_new/v1.1.rst

```diff
@@ -781,6 +781,9 @@ Changelog
   `check_methods_sample_order_invariance` and `check_methods_subset_invariance` tests.
   :pr:`22318` by `Zhehao Liu <MaxwellLZH>`.
 
+- |Fix| :func:`utils.class_weight.compute_class_weight` does not require all
+  in `class_weight` to be in `classes`. :pr:`22595` by `Thomas Fan`_.
```

Comment:
We should update the entry then.

Reply:
I mean to mention that we still raise an error when a class is present in `y` but not in `class_weight`.